pkg/splunk/controller/configmap.go

```diff
@@ -27,7 +28,7 @@ import (
 )
 
 // ApplyConfigMap creates or updates a Kubernetes ConfigMap
-func ApplyConfigMap(client splcommon.ControllerClient, configMap *corev1.ConfigMap) (bool, error) {
+func ApplyConfigMap(client splcommon.ControllerClient, configMap *corev1.ConfigMap, forceUpdate bool) (bool, error) {
```

Comment:
I wish there was a way to pass optional params in go :)

Reply:
Would be nice.  It would make this PR much smaller....